project/src/main/scala/org/broadinstitute/clio/sbt/Docker.scala

```diff
@@ -35,7 +35,17 @@ object Docker {
     new Dockerfile {
       from("openjdk:8")
       label("CLIO_VERSION", version.value)
-      expose(8080)
+      runRaw(
+        "wget -q https://download-keycdn.ej-technologies.com/jprofiler/jprofiler_linux_9_2_1.tar.gz -P /tmp/ &&" +
+          " tar -xzf /tmp/jprofiler_linux_9_2_1.tar.gz -C /usr/local &&" +
+          " rm /tmp/jprofiler_linux_9_2_1.tar.gz"
```

Comment:
Would it be worth sticking `"jprofiler_linux_9_2_1.tar.gz"` in a local val?

Reply:
Definitely